components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -87,136 +100,158 @@ tweaking their $case/namelist_scream.xml file.
+    </atm_proc_group>
+
+    <!-- Basic options for each "physics" atm process -->
+    <physics_proc_base inherit="atm_proc_base">
+      <Grid>Physics GLL</Grid>
+    </physics_proc_base>
+
+    <!-- Homme dynamics -->
+    <homme inherit="atm_proc_base">
+      <Grid>Dynamics</Grid>
+      <Vertical__Coordinate__Filename>${DIN_LOC_ROOT}/atm/scream/init/init_ne4np4.nc</Vertical__Coordinate__Filename>
+      <Vertical__Coordinate__Filename COMPSET=".*SCREAM%AQUA.*">${DIN_LOC_ROOT}/atm/scream/init/scream-aquaplanet_init_ne4np4_L72_20211202.nc</Vertical__Coordinate__Filename>
+      <Moisture>moist</Moisture>
+    </homme>
+
+    <!-- P3 microphysics -->
+    <p3 inherit="physics_proc_base"/>
+
+    <!-- SHOC macrophysics -->
+    <shoc inherit="physics_proc_base"/>
+
+    <!-- CLD fraction -->
+    <cldFraction inherit="physics_proc_base"/>
+
+    <!-- Simple Prescribed Aerosols (SPA) -->
+    <spa inherit="physics_proc_base">
+      <SPA__Remap__File>none</SPA__Remap__File>
+      <SPA__Data__File>${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne4_scream.nc</SPA__Data__File>
+    </spa>
+
+    <!-- Simple Prescribed Aerosols (SPA) -->
+    <rrtmgp inherit="physics_proc_base">
+      <active_gases>h2o, co2, o3, n2o, co, ch4, o2, n2</active_gases>
+      <Orbital__Year>-9999</Orbital__Year>
+      <Orbital__Eccentricity>-9999</Orbital__Eccentricity>
+      <Orbital__Eccentricity COMPSET=".*SCREAM%AQUA.*">0</Orbital__Eccentricity>
+      <Orbital__Obliquity>-9999</Orbital__Obliquity>
+      <Orbital__Obliquity COMPSET=".*SCREAM%AQUA.*">0</Orbital__Obliquity>
+      <Orbital__MVELP>-9999</Orbital__MVELP>
+      <Orbital__MVELP COMPSET=".*SCREAM%AQUA.*">0</Orbital__MVELP>
+    </rrtmgp>
+
+    <mac_aero_mic inherit="atm_proc_group">
+      <atm_procs_list>(shoc,cldFraction,spa,p3)</atm_procs_list>
+    </mac_aero_mic>
+
+    <physics inherit="atm_proc_group">
+      <atm_procs_list>(mac_aero_mic,rrtmgp)</atm_procs_list>
+    </physics>
+  </atmosphere_processes_defaults>
+
+  <!-- Grids manager specs -->
+  <Grids__Manager>
+    <Type>Dynamics Driven</Type>
+    <Reference__Grid>Physics GLL</Reference__Grid>
+    <Dynamics__Namelist__File__Name>./data/namelist.nl</Dynamics__Namelist__File__Name>
+  </Grids__Manager>
+
+  <!-- List of nc files for loading inputs on specified grids -->
+  <Initial__Conditions>
+    <Filename>${DIN_LOC_ROOT}/atm/scream/init/init_ne4np4.nc</Filename>
+    <Filename COMPSET=".*SCREAM%AQUA.*">${DIN_LOC_ROOT}/atm/scream/init/scream-aquaplanet_init_ne4np4_L72_20211202.nc</Filename>
+  </Initial__Conditions>
+
+  <!-- List of yaml files containing I/O output specs -->
+  <Scorpio>
+    <Output__YAML__Files>data/scream_output.yaml, </Output__YAML__Files>
+  </Scorpio>
+
+  <!-- Debug options for scream -->
+  <Debug>
+    <Atmosphere__DAG__Verbosity__Level>0</Atmosphere__DAG__Verbosity__Level>
+    <Atm__Log__Level valid_values="trace,debug,info,warn,error">info</Atm__Log__Level>
```

Comment:
I like the `valid_values` control.

Reply:
@jgfouca idea. It is already in master. I just added a few usages here and there.